tests/ami/scripts/upgrade_install_rpm.sh.j2

```diff
@@ -64,11 +68,19 @@ echo ""
 ls -la /etc/rc*.d/ | grep scalyr-agent | wc -l | grep 7
 
 # Upgrade to new version
-{% if package_to_version_is_url -%}
-wget -O package-to.rpm "{{ package_to_version }}"
-sudo rpm -Uv package-to.rpm
+{% if package_to_version_source_type in ("file", "url") -%}
+    {% if package_to_version_source_type == "url" -%}
+    wget -O to_package.rpm "{{ package_to_version }}"
+    {% endif -%}
+
+
+echo "1"
```

Comment:
I assume you used those echo lines for debugging and they are not needed anymore?

Reply:
Yes.. Thanks for pointing out.